sdk/src/main/java/io/dapr/actors/DaprClientBuilder.java

```diff
@@ -21,10 +22,10 @@
      * Builds an async client.

```

Comment:
Name of client doenst match the Return type of BuildAsyncClient method, the method is returning ActorProxyToAppHttpAsyncClient This class can be package private also as it will be used by ActorProxyonly, (both will be in io.dapr.actors.clientpackage)

Reply:
Make package private and moved.  I think the name of `buildAsyncClient()` method is ok though.  The caller will have an instance of this class to call this and that will show the type.